### ISSUE TITLE
Extensions for Rust and Go not documented

### ISSUE THREAD

**Issue by Kannen** [autozimu/LanguageClient-neovim#1224]
- Did you upgrade to latest plugin version?
- Did you upgrade to/compile latest binary? Run shell command
  `bin/languageclient --version` to get its version number.
- (Neovim users only) Did you check output of `:checkhealth LanguageClient`?
- Did you check [troubleshooting]?

[troubleshooting]: https://github.com/autozimu/LanguageClient-neovim/blob/next/INSTALL.md#troubleshooting

## Describe the bug

This is a bug in the documentation: its said that LanguageClient has extensions for Rust and Go in paragraph §2.40 but it is not said how those extensions are supposed to be run.

For exemple on a rust file I can clearly see on the left of a doctest "=> Run Doctest". But it don't know, and the documentation does not say how to run this test. This should be specified, I suppose this is only 4 words.

**Comment by martskins** [comment]
Yeah, the thing with these is that they are very server specific, so documenting them is tricky. I guess a little more info on codelenses could be included in the documentation though.
I guess the expectation was that the user would know that those are codelenses, and that there is a function to use them (that one is documented, although the documentation for it is probably a bit vague), and all code lenses are used in the same way irregardless of what they do.
Happy to receive any documentation updates on that though, but I wouldn't go too much into the specifics (like listing each and every code lens for both Rust and Go) because as I said those are very server specific, they are not something we control, and their behavior could change or they could even cease to exist.
